ui/apps/everest/.e2e/utils/db-cmd-line.ts

```diff
@@ -292,3 +307,64 @@ export const queryTestDB = async (cluster: string, namespace: string) => {
   }
   return result;
 };
+
+export const prepareMongoDBTestDB = async (
+  cluster: string,
+  namespace: string
+) => {
+  await dropTestDB(cluster, namespace);
+
+  // Insert test data into the collection t1 and t2
+  console.log('Preparing test database for MongoDB...');
+  await queryPSMDB(
+    cluster,
+    namespace,
+    'test',
+    'db.t1.insertMany([{ a: 1 }, { a: 2 }, { a: 3 }]);'
+  );
+
+  await queryPSMDB(
+    cluster,
+    namespace,
+    'test',
+    'db.t2.insertMany([{ b: 4 }, { b: 5 }, { b: 6 }]);'
```

Comment:
We could have used the same collection structure, just different collection name, to simplify things eg: `db.t2.insertMany([{ a: 1 }, { a: 2 }, { a: 3 }]);'`. But I will leave it to you.

Reply:
I agree with that, but I decided to keep it as `a` and `b` to see how sharding was distributing between the shards and to easily identify which collections they belong to.